src/bulk/ordered.ts

```diff
@@ -26,7 +26,7 @@ export class OrderedBulkOperation extends BulkOperationBase {
 
     // Throw error if the doc is bigger than the max BSON size
     if (bsonSize >= this.s.maxBsonObjectSize)
-      throw new MongoDriverError(
+      throw new MongoInvalidArgumentError(
```

Comment:
maxBsonObjectSize is a property from the server, I believe it can be bumped or reduced in config. But regardless the check at this point determines if the JS object passed in would serialize to something that exceeds the maximum so I think InvalidArgument does apply. 

We could also consider making a MongoBSONError right now and thinking about subclasses for that later. Could cover `MongoBSONSerializationError, MongoBSONDeserializationError, MongoBSONSizeError`